man/fabric.7.md

```diff
@@ -238,6 +238,50 @@ FI_LOG_SUBSYS environment variables.
 - *mr*
 : Provides output specific to memory registration.
 
+# FILE SYSTEM CONFIGRATION INTERFACE
+
+Libfabric defines a set of file system interfaces that may be used to
+set global or provider specific parameters.  Conceptually, this interface
+is similar to the /proc interfaces used by ip.7 and tcp.7.
+Parameters can be accessed by reading or writing files in
+%SYSCONFDIR%/rdma/fabric.  The files are typically found at
+/etc/rdma/fabric or /usr/local/etc/rdma/fabric.
+The exact location of the files is controllable through build parameters,
+but for simplicity, the following documentation assumes that they are found
+under /etc/rdma/fabric.  The files do not exist by default and must be
+created by the administrator or Linux distro.  
+
+Parameter files located under /etc/rdma/fabric are treated as global and
+apply to all relevent providers.  Provider specific parameters may be set
+by writing to a file by the same name, but in a provider specific
+subdirectory.  For example, parameter files under /etc/rdma/fabric/verbs
+will apply only to the verbs provider.
+
+Note that providers may select alternate values that those
+specified.  This may be necessary in order to adhere to memory alignment
+restrictions, hardware maximums, or protocol limitations, for example.
+
+*conn_timeout - Connection Request Timeout (integer)*
+: Specifies the total time, in milliseconds, before a connection operation
+  times out.
+
+*def_inject_size - Default Inject Size (integer)*
```

Comment:
This gets weird, though.  What if a valid is invalid for provider A, but valid for provider B?  Should provider A log a warning/error, but then provider B succeed and run properly?

BTW, should it also be noted exactly when these values are read?  You raise a good point that we don't have the ability to read the values exactly when they are read (like the kernel) -- so it would be good to set the expectation that if you start a long libfabric-based job, if you write a new value to one of these files during the middle of the run, it won't change anything in the already-running job (right?).